js/repl.ts

```diff
@@ -71,6 +80,19 @@ export async function replLoop(): Promise<void> {
     }
     if (!code) {
       continue;
+    } else if (code.trim() === ".help") {
+      console.log(help);
+      continue;
+    } else if (code.trim().startsWith(".load")) {
```

Comment:
I am pretty sure we don't want to load code this way.  It will take a bit more work, of the REPL oping out to Rust to load the module in a similar fashion to the way modules are solved now.

Reply:
@kitsonk thank you for your feedback! Why do you think we should load files in repl like modules? Files can contain only separate functions or even expressions, so in some cases, it can't be resolved as a module.

For example, in node.js repl checks if loaded path is a file and after that reads a file in a similar way

```javascript
  repl.defineCommand('load', {
    help: 'Load JS from a file into the REPL session',
    action: function(file) {
      try {
        var stats = fs.statSync(file);
        if (stats && stats.isFile()) {
          _turnOnEditorMode(this);
          var data = fs.readFileSync(file, 'utf8');
          var lines = data.split('\n');
          for (var n = 0; n < lines.length; n++) {
            if (lines[n])
              this.write(`${lines[n]}\n`);
          }
          _turnOffEditorMode(this);
          this.write('\n');
        } else {
          this.outputStream.write('Failed to load: ' + file +
                                  ' is not a valid file\n');
        }
      } catch {
        this.outputStream.write('Failed to load: ' + file + '\n');
      }
      this.displayPrompt();
    }
  });
```